build_runner/lib/src/entrypoint/options.dart

```diff
@@ -399,3 +419,36 @@ Set<BuildDirectory> _parseBuildDirs(ArgResults argResults) {
   }
   return result;
 }
+
+/// Returns build filters parsed from [buildFilterOption] arguments.
+///
+/// These support `package:` uri syntax as well as regular path syntax,
+/// with glob support.
+List<BuildFilter> _parseBuildFilters(
+    ArgResults argResults, String rootPackage) {
+  var filterArgs = argResults[buildFilterOption] as List<String>;
+  if (filterArgs?.isNotEmpty != true) return null;
```

Comment:
[nit] https://dart.dev/guides/language/effective-dart/usage#do-use--to-convert-null-to-a-boolean-value